index.js

```diff
@@ -157,10 +157,59 @@ function displayFooter() {
 }
 
 function activateSortBtns() {
-  sortBtns.forEach((btn) => btn.addEventListener("click", handleSortBtnClick));
+  sortBtns.forEach((btn) => {
+    btn.addEventListener("click", handleSortBtnClick);
+    btn.sortingState = null;
+  });
 
   function handleSortBtnClick(e) {
-    console.log(e.target.value);
+    let sortedPosts;
+    sortBtns.forEach((btn) => {
+      if (btn.value !== e.target.value) {
+        btn.sortingState = null;
+      }
+    });
+    let [sortBtn, sortedBy] = [e.target, e.target.value];
+    postsContainer.innerHTML = "";
+    if (
+      !sortBtn.sortingState ||
+      sortBtn.sortingState === "onEventhInARowClick"
```

Comment:
and if you use variables like `const ASCENDING_ORDER = 0, DESCENDING_ORDER = 1;` you can write:
```suggestion
      currentSortOrder === DESCENDING_ORDER
```

Reply:
that can be done

Reply:
Are you going to complete this so it can be merged?

Reply:
I will work on that, presumably today